protocols/kad/src/handler.rs

```diff
@@ -803,17 +808,19 @@ impl ConnectionHandler for KademliaHandler {
                     match (remote_supports_our_kademlia_protocols, self.protocol_status) {
                         (true, ProtocolStatus::Confirmed | ProtocolStatus::Reported) => {}
                         (true, _) => {
-                            log::info!(
-                                "Remote {} now supports our kademlia protocol",
-                                self.remote_peer_id
+                            log::debug!(
+                                "Remote {} now supports our kademlia protocol. {:?}",
```

Comment:
Can we make `ConnectionId` implement `fmt::Display` and make this log-message a bit prettier?

```suggestion
                                "Remote {} now supports our kademlia protocol on connection {}",
```

Reply:
What output message is preferable for `impl Display for ConnectionId` ?

Reply:
I think just the number is good enough.

Reply:
I didn't update the swarm change log (tiny change - impl Display for ConnectionId).